CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [next-version]
 
+### Changed
+
+- Removed `onDocumentCapture` that used to be fired when the document had been successfully captured, confirmed by the user and uploaded to the Onfido API
```

Comment:
Minor: I think we have a removed section.

Reply:
Fixed